Spectrum_Peak_Visualization.py

```diff
@@ -54,90 +55,89 @@ def plot_peaks(spectrum, title= None, energy_range= None, subregion='both', use=
-             
+
     if subregion == 'left':
-    
-        for i in range(int(len(center_peak_region)/2)):
-            plt.fill_between(energy_axis, counts, where=(energy_axis >= 
-                             energy_axis[center_peak_region[2*i]]) &
-                             (energy_axis <= 
-                             energy_axis[center_peak_region[2*i+1]]) , 
+
+        for i in range(int(len(center_peak_region) / 2)):
+            plt.fill_between(energy_axis, counts, where=(energy_axis >=
+                             energy_axis[center_peak_region[2 * i]]) &
+                             (energy_axis <=
+                             energy_axis[center_peak_region[2 * i + 1]]),
                              facecolor='r')
-            plt.fill_between(energy_axis, counts, where=(energy_axis >= 
-                             energy_axis[left_peak_region[2*i]]) &
-                             (energy_axis <= 
-                             energy_axis[left_peak_region[2*i+1]]), 
+            plt.fill_between(energy_axis, counts, where=(energy_axis >=
+                             energy_axis[left_peak_region[2 * i]]) &
+                             (energy_axis <=
+                             energy_axis[left_peak_region[2 * i + 1]]),
                              facecolor='g')
-                             
+
     if subregion == 'right':
-        
-        for i in range(int(len(center_peak_region)/2)):
-            plt.fill_between(energy_axis, counts, where=(energy_axis >= 
-                             energy_axis[center_peak_region[2*i]]) &
-                             (energy_axis <= 
-                             energy_axis[center_peak_region[2*i+1]]) , 
+
+        for i in range(int(len(center_peak_region) / 2)):
+            plt.fill_between(energy_axis, counts, where=(energy_axis >=
+                             energy_axis[center_peak_region[2 * i]]) &
+                             (energy_axis <=
+                             energy_axis[center_peak_region[2 * i + 1]]),
                              facecolor='r')
-            plt.fill_between(energy_axis, counts, where=(energy_axis >= 
-                             energy_axis[right_peak_region[2*i]]) &
-                             (energy_axis <= 
-                             energy_axis[right_peak_region[2*i+1]]), 
+            plt.fill_between(energy_axis, counts, where=(energy_axis >=
+                             energy_axis[right_peak_region[2 * i]]) &
+                             (energy_axis <=
+                             energy_axis[right_peak_region[2 * i + 1]]),
                              facecolor='g')
-        
-    
+    if energy_range is not None:
+        plt.xlim([energy_range[0], energy_range[1]])
     plt.yscale('log')
     plt.xlabel('Energy (keV)')
     plt.ylabel('Counts')
-    plt.show()
-
-    if energy_range is not None:
-        plt.xlim([energy_range[0], energy_range[1]])
+    plt.savefig(str(title) + '.png')
```

Comment:
Instead of putting this here in `spv.plot_peaks()`, put it in `Gamma_Analysis.save_peaks()`. We don't want to save a PNG file every time we visualize peaks.